style.css

```diff
@@ -0,0 +1,114 @@
+@import url("https://fonts.googleapis.com/css2?family=Poppins:wght@300&display=swap");
+
+body {
+  display: flex;
+  flex-direction: column;
+  background-color: #cac4c4;
```

Comment:
According to the changes required by the previous reviewer, I highly suggest you make sure that the body of the page has the same background color compared to the template, I recommend using this one `#E5E5E5`. 😄